### ISSUE TITLE
AttributeError: function 'get_osfhandle' not found

### ISSUE THREAD

**Issue by smasato** [nfcpy/nfcpy#135]
Running `examples/tagtool.py` on Windows 10 will generate an error.

```
C:\Users\***\***\nfcpy-master\nfcpy-master\examples>tagtool.py
Traceback (most recent call last):
  File "C:\Users\***\***\nfcpy-master\nfcpy-master\examples\tagtool.py", line 625, in <module>
    TagTool().run()
  File "C:\Users\***\***\nfcpy-master\nfcpy-master\examples\tagtool.py", line 295, in __init__
    super(TagTool, self).__init__(parser, groups="rdwr card dbg clf")
  File "C:\Users\***\***\nfcpy-master\nfcpy-master\examples\cli.py", line 85, in __init__
    ch = ColorStreamHandler()
  File "C:\Users\***\***\nfcpy-master\nfcpy-master\examples\cli.py", line 501, in __init__
    self._outhdl = crtlib.get_osfhandle(self.stream.fileno())
  File "C:\Python27\lib\ctypes\__init__.py", line 379, in __getattr__
    func = self.__getitem__(name)
  File "C:\Python27\lib\ctypes\__init__.py", line 384, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: function 'get_osfhandle' not found
```

I examined this lines.
https://github.com/nfcpy/nfcpy/blob/b6a4904fe8fb1f568966cef8d61d38f592db90c9/examples/cli.py#L493-L501
```
Python 2.7.16 (v2.7.16:413a49145e, Mar  4 2019, 01:37:19) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import ctypes.util
>>> import ctypes
>>> crtname = ctypes.util.find_msvcrt()
>>> crtname
'msvcr90.dll'
>>> crtlib = ctypes.cdll.LoadLibrary(crtname)
>>> crtlib.get_osfhandle
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python27\lib\ctypes\__init__.py", line 379, in __getattr__
    func = self.__getitem__(name)
  File "C:\Python27\lib\ctypes\__init__.py", line 384, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: function 'get_osfhandle' not found
>>> crtlib
<CDLL 'msvcr90.dll', handle 70040000 at 3317a90>
>>> crtlib._get_osfhandle
<_FuncPtr object at 0x00000000033765F8>
```

As a result, I thought that the function name is `_get_osfhandle` instead of `get_osfhandle`.
And when I fixed `examples/tagtool.py` like this, it worked fine.
```diff
-     self._outhdl = crtlib.get_osfhandle(self.stream.fileno()) 
+     self._outhdl = crtlib._get_osfhandle(self.stream.fileno()) 
```

Is this an error only for me?

**Comment by nehpetsde** [comment]
@smasato Thanks for reporting. Your observation is consistent with all top-level search results - they all indicate that `_get_osfhandle` is the correct function name.

**Comment by nehpetsde** [comment]
The bug was accidentally introduced with release 1.0.0 on June 7 as part of the Python2 to Python3 effort and is now fixed with release 1.0.3.
